src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/UnsupportedTypeConverterFactory.cs

```diff
@@ -23,6 +23,8 @@ public override bool CanConvert(Type type)
                 type == typeof(SerializationInfo) ||
                 type == typeof(IntPtr) ||
                 type == typeof(UIntPtr) ||
+                // Exlude delegates.
+                typeof(Delegate).IsAssignableFrom(type) ||
```

Comment:
probably more readable
```suggestion
                type.IsAssignableTo(typeof(Delegate)) ||
```

Reply:
I prefer the former which is already prevalent in the STJ codebase.